archivers/libzzip/Portfile

```diff
@@ -66,12 +67,10 @@ post-destroot {
 variant sdl description {Enable SDL support} {
     depends_lib-append      port:libsdl
 
-    configure.args-append   --enable-sdl
+    configure.args-delete   -DZZIPSDL=OFF
+    configure.args-append   -DZZIPSDL=ON
 
     post-destroot {
         xinstall -m 0644 ${worksrcpath}/docs/README.SDL ${destroot}${docdir}
     }
 }
-
-test.run        yes
-test.target     check
```

Comment:
can we keep the tests enabled?

Reply:
@reneeotten well... with migration to `cmake` they run tests during the build, and it failed with a error:
```
dyld: Library not loaded: /opt/local/lib/libzzip.13.dylib
  Referenced from: /opt/local/var/macports/build/_Users_catap_src_macports-ports_archivers_libzzip/libzzip/work/build/test/./zzipsetstub
  Reason: image not found
```

Can I fix it? Prehabs. Bug I feel it as waste of time. A lot of time.

Reply:
Tests are fixed.